### PR TITLE
converted tests to py.test

### PR DESCRIPTION
**Pull request by ErikPel**
Still work in progress. Everything else seems to work except for 
test_enable_app_disable_app
test_update_share_user
test_get_shares_empty 
test_share_with_link

Maybe someone can take a look and help me figure out whats wrong with them

### CODE REVIEW

File: owncloud/test/test.py [hunk 1]
```diff
@@ -1,20 +1,18 @@
 # -*- coding: utf-8 -*-
-# vim: expandtab shiftwidth=4 softtabstop=4
```
Comment (PVince81) [inline owncloud/test/test.py]: how dare you remove my vim settings :wink:

Comment (ErikPel) [inline owncloud/test/test.py]: It was an accident, I swear! :anguished:

**Comment by ErikPel** [comment]
Seems like the tests are failing on owncloud 9 but I can't figure out why.

**Comment by ErikPel** [comment]
Got all the tests working so if you end up using this it's ready for review. But if not, no harm done, it was a good learning experience.
@PVince81

**Comment by PVince81** [comment]
Thanks a lot, I haven't made a decision yet.

You're right that having both py.test and unittest would require two dependencies, which isn't that good.

**Comment by jamescooke** [comment]
What was your decision @PVince81 ?

**Comment by PVince81** [comment]
I'd prefer to stay with the old style for now.

**Pull request closed without merging**